six/modules/c++/cphd/unittests/test_pvp_block_round.cpp

```diff
@@ -0,0 +1,413 @@
+/* =========================================================================
+ * This file is part of cphd-c++
+ * =========================================================================
+ *
+ * (C) Copyright 2004 - 2019, MDA Information Systems LLC
+ *
+ * cphd-c++ is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; If not,
+ * see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include <cstdio>
+#include <stdlib.h>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <memory>
+#include <cphd/Wideband.h>
+#include <cphd/Metadata.h>
+#include <cphd/PVP.h>
+#include <cphd/PVPBlock.h>
+#include <cphd/CPHDWriter.h>
+#include <cphd/CPHDReader.h>
+#include <cphd/ReferenceGeometry.h>
+#include <types/RowCol.h>
+#include <cli/ArgumentParser.h>
+#include <io/TempFile.h>
+#include <io/FileInputStream.h>
+#include <io/FileOutputStream.h>
+#include <TestCase.h>
+
+namespace
+{
+double getRandom()
```

Comment:
Can you extract the common stuff like this to some `testDataGenerator.h` header?